openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -24,53 +22,45 @@ import '../../../../vendor/js/flot/jquery.flot.stack.js';
 import '../../../../vendor/js/flot/jquery.flot.pie.js';
 import { validateEmail, validatePassword } from './account.js';
 import autocompleteInit from './autocomplete';
+// Used only by the openlibrary/templates/books/edit/addfield.html template
 import addNewFieldInit from './add_new_field';
 import automaticInit from './automatic';
-import { getAvailabilityV2,
-    updateBookAvailability, updateWorkAvailability } from './availability';
 import bookReaderInit from './bookreader_direct';
 import Carousel from './carousels';
 import { ungettext, ugettext,  sprintf } from './i18n';
 import addFadeInFunctionsTojQuery from './jquery.customFade';
 import jQueryRepeat from './jquery.repeat';
-import './jquery.scrollTo';
 import { enumerate, htmlquote, websafe, foreach, join, len, range } from './jsdef';
 import { plot_minigraph, plot_tooltip_graph } from './plot';
 import initAnalytics from './ol.analytics';
-import init, { closePop, bookCovers, isScrolledIntoView } from './ol.js';
```

Comment:
`window.bookCovers` is still used :( Doing a whole word search reveals results:

![image](https://user-images.githubusercontent.com/6251786/64910698-6746e680-d6e7-11e9-8d3e-8a6643a7dca2.png)

Reply:
good save!

Reply:
Oh wait. Now I remember this. This code doesn't actually do anything... let me remove from those templates instead.

Reply:
I've raised this - https://github.com/internetarchive/openlibrary/pull/2397 - yet more dead code to remove.

the bookCovers function used to provide the functionality described in https://github.com/internetarchive/openlibrary/issues/2247 but since we moved to Slick this code doesn't seem to be used anywhere. It's frustrating there's not a quicker way to discover dormant templates. I really think writing some code to test these programatically would be a productive use of time as this slows us down a lot with the frontend.

Reply:
What about `CoverImageWork`, `CoverImageAuthor`, and `SearchResultCovers`? They all use this. Can we leave it in for now and remove it in a future issue? That'll make this a lot faster to go through. Why do you say it doesn't do anything?